### PR TITLE
Remove mailing list references

### PR DESCRIPTION
**Pull request by jdaugherty**

### CODE REVIEW

File: conf/questions.yml [hunk 1]
```diff
@@ -26,15 +26,15 @@ questions:
   -
     slug: 'question_mailinglist'
     title: 'Do you have a mailing list?'
-    answer: 'Yes. Subscribe to our Google Group [here](https://groups.google.com/forum/#!forum/grails-dev-discuss).'
+    answer: 'No, we prefer to converse over GitHub, Slack, or Stack Overflow.'
```
Comment (sbglasius) [inline conf/questions.yml:29]: This will be true until we move to ASF

File: conf/questions.yml [hunk 2]
```diff
@@ -26,15 +26,15 @@ questions:
...
   -
     slug: 'question_feature'
     title: 'Where can I suggest a new feature or share an interesting finding?'
-    answer: 'The [Grails mailing list](https://groups.google.com/forum/#!forum/grails-dev-discuss) is a great place to start a conversation with the Grails developers and other Grails users.'
+    answer: 'The [Grails Community Slack](https://slack.grails.org) is a great place to start a conversation with the Grails developers and other Grails users.'
```
Comment (jeffscottbrown) [inline conf/questions.yml:33]: > The [Grails Community Slack](https://slack.grails.org) is a great place to start a conversation with the Grails developers and other Grails users.

This is a response to "Where can I suggest a new feature or share an interesting finding?".  I don't think Stackoverflow is a good place for that.  I would suggest https://github.com/orgs/grails/discussions.

**Review by jeffscottbrown — CHANGES_REQUESTED** [review]

File: conf/questions.yml (hunk 1 above)
Comment (jamesfredley) [reply conf/questions.yml:29]: I'd reorder these to 

`answer: 'No, we prefer to converse over Slack, GitHub or Stack Overflow.'`

and if we want to give users a bit of a preview of the future:

`answer: 'No, we prefer to converse over Slack, GitHub or Stack Overflow.  Mailing lists are planned after Grails transition to ASF'`

**Review by jamesfredley — COMMENTED** [review]
Just the one comment on mailing list.

File: conf/questions.yml (hunk 1 above)
Comment (jdaugherty) [reply conf/questions.yml]: I updated this to James's second answer.  Is this acceptable?

**Review by jamesfredley — APPROVED** [review]

**Pull request merged**